.github/workflows/buildAndDeployPhar.yml

```diff
@@ -0,0 +1,46 @@
+# https://help.github.com/en/categories/automating-your-workflow-with-github-actions
+
+name: Build and deploy PHAR
+on:
+  push:
+    tags:
+    - "*"
+jobs:
+  build:
+    name: Build PHAR-File
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@master
+
+    - name: Build Phar
+      uses: MilesChou/composer-action/7.2/install@master
+      with:
+        args: buildphar
+
+    - name: Sign Phar
+      env:
+        DECRYPT_KEY: ${{ secrets.DECRYPT_KEY }}
+        SIGN_KEY: ${{ secrets.SIGN_KEY }}
+      run: |
+        export GPG_TTY=$(tty)
+        echo $DECRYPT_KEY
+        echo Decrypting key
+        gpg --batch --yes --passphrase $DECRYPT_KEY KeepAChangelogSigning.asc.gpg && gpg --batch --yes --import KeepAChangelogSigning.asc
+        echo Signing Artifact
+        gpg --batch --yes --pinentry-mode loopback --passphrase $SIGN_KEY -u 9A2577FF9A688FAF --armor --detach-sig build/keep-a-changelog.phar
+
+    - name: Prepare Release-Info
+      run: |
+        echo ::set-env name=RELEASE_VERSION::${GITHUB_REF/refs\/tags\//}
+        ./build/keep-a-changelog.phar version:show ${{ env.RELEASE_VERSION }} > release.note
+
+    - name: Create release
+      uses: ncipollo/release-action@v1
+      with:
+#        artifacts: "build/*"
```

Comment:
Just noticed this... because I discovered that the asset wasn't built!

Going to delete the 2.3.0 tag, update this, and re-tag, and we should have the asset finally.